data.py

```diff
@@ -30,14 +30,31 @@ class DatasetFormat:
     XSUM_SUMMARIZATION: str = "xsum_summarization"
     HUMAN_EVAL: str = "human_eval"
     CUSTOM_JSONL: str = "custom_jsonl"
+    TOP_v2: str = "top_v2"
```

Comment:
I suggest to use all uppercase to be consistent with the used enum convention
```suggestion
    TOP_V2: str = "top_v2"
```